assets/js/common/session.js

```diff
@@ -36,6 +37,8 @@ function handleSessionExpiration() {
             isAuthenticated = false;
             clearSessionExpiryWarningTimer();
             trackEvent('Session', 'Warning', 'Timeout reached');
+            // Prevent the page abandonment warning from showing
+            forms.removeBeforeUnload();
```

Comment:
This is a touch awkward, suggests this session code should probably live in `common/forms.js` instead.

Reply:
Done!